src/lib/components/shared/text_editor.svelte

```diff
@@ -406,14 +414,24 @@
         <div class="flex place-items-center gap-2 pr-4 md:gap-[0.75vw] md:pr-[1vw]">
             {#each Object.entries(icon_and_function_mapping) as item}
                 {@const item_label = item[0]}
+
                 {@const icon = item[1].icon.component}
                 {@const icon_class = item[1].icon.class}
                 {@const button_function = item[1].function}
+                {@const description = item[1].description}
 
                 <button
                     class="btn p-0"
                     type="button"
                     aria-label={item_label}
+                    use:tippy={{
+                        content: `<div class='bg-surface-400 px-3 py-1 text-surface-50 text-sm leading-2 rounded-lg'>${description}</div>`,
```

Comment:
Add `vw` based measures ?
And not looking good in mobile, change to `text-xs` and in mobile add `md:text-[1vw]` ig

Reply:
Hmmm